spec/models/user_spec.rb

```diff
@@ -4,6 +4,12 @@
 MAX_GOOD_PASSWORD = '!1aZ' * 32
 
 describe User do
+  describe 'Associations' do
+    it { should have_many(:authorizations) }
```

Comment:
Even though `should` reads better here, I think we should probably be consisent and use the new style `expect` everywhere, no?